docs/src/administration/cli/api-tokens.md

```diff
@@ -1,80 +1,101 @@
-while checking out code from a CI server to run tests on it would require an SSH key but only `Viewer` access.
+To create a machine user, follow the steps:
 
-It also shows up in logs and activity streams as a separate entry from human users.
+1. Run the following command using your machine user's email address.
+   ```bash
+   platform user:add email@example.com
+   ```
+2. Choose the project where you want to add the machine user.
+3. To make the machine user a viewer of the project, press **Enter**.
+4. Grant the user [permissions](/administration/users.md) depending on your needs and the environment type.
+5. To send an invitation to the email address provided in step 1, press **Enter**. 
+6. In the email invitation, click **Create account**.
+7. To create a Platform.sh account for the machine user, follow the steps.
 
-To add the user:
+## 2. Assign an API token to your machine user
 
-1. In a terminal, run `platform user:add email@example.com`, replacing the email with the one for your machine user.
-1. (If not within a specific project) choose a project to add the user to.
-1. Press `enter` to make the user a viewer of the entire project.
-1. Assign the user the correct permissions for each environment type.
-   (See the [users documentation](/administration/users.md) for more on access levels.)
-1. Press `enter` to send the invitation.
-1. In your email, click the link in the invitation to accept and then follow the steps to create an account.
+### Create a Platform.sh API token
 
-## Get a token
-
-Once you have a machine user in place, you want to assign an API token to it.
-
-To get an API token:
-
-1. As the machine user, open the Console.
-2. Open the user menu (your name or profile picture).
+1. Log in to the Console as your machine user.
+2. To open the user menu, click the downward arrow at the top right-hand side of the screen:
+   ![The user menu in the Console](/images/management-console/user-menu.png "0.6")
 3. Click **My profile**.
-4. Go to the **API Tokens** tab.
+4. Go to the **API TOKENS** tab.
 5. Click **Create API Token**.
-   ![The Create API Token button in the Console](/images/management-console/api-tokens-new.png "0.6")
-6. Enter a name to identify your token in the future if you have multiple tokens.
+   ![The Create API Token button in the Console](/images/management-console/create-api-token-button.png "0.6")
+6. Enter a name for your API token and click **Create API Token**.
    ![Creating an API token with the name 'CI tests'](/images/management-console/api-tokens-name.png "0.6")
-7. Click **Copy** to copy the token to your clipboard.
-   Make sure to store the key safely as you can't view the API token again.
+7. To copy the API token to your clipboard, click **Copy**.
    ![Viewing the API token after it's created](/images/management-console/api-tokens-view.png "0.6")
+8. Store the API token somewhere secure on your computer.
+   Note that after you quit the Console, you can't display the API token again.
```

Comment:
Hmm. I thought I remembered it deleting otherwise, but I guess not. Most other security-aware services do this. Guess just another mistake of mine this time.